x-pack/plugins/siem/public/containers/errors/index.tsx

```diff
@@ -5,29 +5,33 @@
  */
 
 import { onError } from 'apollo-link-error';
+import { throttle, noop } from 'lodash/fp';
 import uuid from 'uuid';
 
+import * as i18n from './translations';
+
 import { getStore } from '../../store';
 import { appActions } from '../../store/actions';
 
-import * as i18n from './translations';
-
 export const errorLink = onError(({ graphQLErrors, networkError }) => {
   const store = getStore();
+  const dispatch = throttle(50, store != null ? store.dispatch : noop);
```

Comment:
Why the throttle/noop here? Is the store not initialized when the first errors can come in?

Reply:
Yes, it is still the same problem as before since errorLink is working as almost a middleware